content/Guides/DistributedComplianceLedger.md

```diff
@@ -0,0 +1,142 @@
+
+### Types of nodes that operate in the DCL
+
+| Type of Node         | Description                                                                                                                                          |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Validator Node (VN)  | It participates in creating a "consensus" process to add information into the ledger.                                                                |
+| Sentry Node (SN)     | It doesn\'t participate in consensus and wraps the VN representing it for the rest of the network as one of the ways for DDoS and access protection. |
+| Observer Node (ON)   | It doesn\'t participate in consensus and is optimized for public data read and authenticated write interactions with the ledger.                     |
+
+### Types of account roles that can interact with the DCL
+
+| Account Role         | Description                                                                                                                                          |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Trustee              | It is entrusted to approve or deny other DCL roles and PAA Root CAs in the ledger, as well as to disable Validator Nodes.                            |
+| Node Admin           | It can instantiate a single Validator Node.                                                                                                          |
+| Vendor               | It can write vendor and product information to the ledger. Alliance members can have one or more vendor accounts.                                    |
+| Vendor Admin         | It can add and update the Vendor information table of the ledger.                                                                                    |
+| Certification Center | It can submit, update, delete or revoke certification status of a product based on the Certification application in the Certification Tool.          |
+
+### Main-Net and Test-Net
+
+| DCL Deployment       | Description                                                                                                                                                |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Main-Net             |  It is the DCL utilized by the public and is regarded as the production environment. Transactions on the Main-Net involve real products and data.          |
+| Test-Net             |  It is used for testing and development purposes. It provides a safe environment for experimentation and troubleshooting before deploying to the Main-Net. |

```

Comment:
Drop the "it is" in the descriptions, and maybe reorder the first one

"The production environment that is utilized by the public."
"Used for testing and development purposes..."

Reply:
Done